bluepysnap/nodes.py

```diff
@@ -453,22 +668,24 @@ def get(self, group=None, properties=None):
         """Node properties as a pandas Series or DataFrame.
 
         Args:
-            group (int/sequence/str/mapping/None): Which nodes will have their properties
-                returned depends on the type of the ``group`` argument:
+            group (int/CircuitNodeIds/sequence/str/mapping/None): Which nodes will have their
+            properties returned depends on the type of the ``group`` argument:
 
                 - ``int``: return the properties of a single node.
+                - ``CircuitNodeIds`` return the properties from a NodeCircuitNodeIds.
                 - ``sequence``: return the properties from a list of node.
                 - ``str``: return the properties of nodes in a node set.
                 - ``mapping``: return the properties of nodes matching a properties filter.
                 - ``None``: return the properties of all nodes.
 
-            properties (set): If specified, return only the properties in the set.
+            properties (list): If specified, return only the properties in the list.
```

Comment:
why not all sequences ?

Reply:
because we call the properties directly from the dataframe so tuple fails.